ledger/ledger-api-client/src/main/scala/com/digitalasset/ledger/client/services/admin/UserManagementClient.scala

```diff
@@ -0,0 +1,111 @@
+package com.daml.ledger.client.services.admin
+
+import com.daml.ledger.api.domain
+import com.daml.ledger.api.domain.{UserId, User, UserRight}
+import com.daml.ledger.api.v1.admin.user_management_service.UserManagementServiceGrpc.UserManagementServiceStub
+import com.daml.ledger.api.v1.admin.user_management_service.{GetUserRequest, Right}
+import com.daml.ledger.api.v1.admin.{user_management_service => proto}
+import com.daml.ledger.client.LedgerClient
+import com.daml.ledger.client.services.admin.UserManagementClient.{fromApiRight, fromProtoUser, toApiRight}
+import com.daml.lf.data.Ref
+import com.daml.lf.data.Ref.Party
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final class UserManagementClient
+  (service: UserManagementServiceStub)
+  (implicit ec: ExecutionContext)
```

Comment:
To save time, run `./fmt.sh --test --diff && ./ci/check-changelog.sh` on your machine before pushing. Or you can add it to a pre-push hook.

Reply:
Thx. Will do.